### DISCUSSION TITLE
Production and Sandbox harvesting consoles maintence tasks - Tuesday 3 December 2024 from 15:30 CET

### DISCUSSION THREAD

**Discussion by jescriu** [INSPIRE-MIF/helpdesk-geoportal#236] [Announcements]
Dear INSPIRE Geoportal users,

The service of the [production harvesting console](https://inspire-geoportal.ec.europa.eu/harvestconsole) and the [sandbox console](https://inspire-geoportal.ec.europa.eu/sandboxconsole) will be disabled this afternoon from 15:30 CET to perform maintenance tasks for the deployment of a (second) updated version of the Link-Checker. 

This update incorporates a complete implementation of the [Data-Service-Linking-Simplification](https://github.com/INSPIRE-MIF/gp-data-service-linking-simplification) and the correction of bugs already implemented in the first update made last week (affecting the processing of the protocol field + relaxed checks for matching INSPIRE identifiers between the services and the dataset/series metadata files).

The intervention is expected to last a maximum of 2 hours. We will update this notification when the service is available again. Apologies for any inconveniences caused.

**Comment by manilly** [comment]
Dear @jescriu,
you announced enhancements regarding the checks for accessibility of datasets with the last 2 new versions of geoportal. My question is: Are  these checks the same as in the mr-tools (https://github.com/INSPIRE-MIF/mr-tools) since the last commit there is 6 months ago.
Or are the checks in Geoportal different ones?
Thanks, Manuel

**Comment by jescriu** [comment]
Dear @manilly,
The Link-Checker enhancements to implement the Data-Service Linking Simplification good practice, incorporate the same approaches used in the mr-tools (https://github.com/INSPIRE-MIF/mr-tools) into the GeoNetwork Link-Checker.
